package.json

```diff
@@ -82,6 +82,7 @@
     "ember-metrics": "^0.12.1",
     "ember-models-table": "^2.5.0",
     "ember-moment": "^7.7.0",
+    "ember-multiselect-checkboxes": "^0.11.1",
```

Comment:
Do we really need an addon for something as simple and straightforward as checkboxes ?